build.go

```diff
@@ -64,7 +64,8 @@ func buildPlugins(ctx *Context) {
 	}
 	must(CorePlugins.installPackages(plugins...))
 	for _, plugin := range plugins {
-		plugin, err := CorePlugins.ParsePlugin(strings.Split(plugin, "@")[0])
+		s := strings.Split(plugin, "@")
```

Comment:
it's not necessary here because the package.json would be invalid, but I added it anyway